core/deployment/src/main/java/io/quarkus/deployment/pkg/steps/NativeImageBuildStep.java

```diff
@@ -769,9 +769,10 @@ private static void objcopy(String... args) {
             static final Version VERSION_20_1 = new Version("GraalVM 20.1", 20, 1, Distribution.ORACLE);
             static final Version VERSION_20_2 = new Version("GraalVM 20.2", 20, 2, Distribution.ORACLE);
             static final Version VERSION_20_3 = new Version("GraalVM 20.3", 20, 3, Distribution.ORACLE);
+            static final Version VERSION_21_0 = new Version("GraalVM 21.0", 21, 0, Distribution.ORACLE);
 
             static final Version MINIMUM = VERSION_20_1;
```

Comment:
I think I would make 20.3 the minimal version and drop the `graalVmWorkaround()`.

Reply:
I thought about it as well, I am actually working on dropping 20.1 right now :)
I am +1 for dropping 20.2 as well!

Reply:
That being said, it can be done in another PR if you prefer.

Reply:
Hmm, I think it's OK to have them all in this PR.

Reply:
Ready :)
Should I click "Ready for review"?

Reply:
Yes, sure, looks ready to me.